packages/etherlime/deployed-contract/deployed-contract-wrapper.js

```diff
@@ -1,8 +1,5 @@
 const ethers = require('ethers');
-const isAddress = require('./../utils/address-utils').isAddress;
-const isSigner = require('./../utils/signer-utils').isSigner;
-const isValidContract = require('./../utils/contract-utils').isValidContract;
-const colors = require('./../utils/colors');
+const {isAddress, isSigner, isValidContract, colors} = require('etherlime-utils')
```

Comment:
Half the places have spaces around the curly brackets and half of them do not, please stick with spaces around the curly brackets